test/e2e/config/azure-dev.yaml

```diff
@@ -196,4 +196,8 @@ intervals:
   default/wait-job: ["5m", "10s"]
   default/wait-service: ["15m", "10s"]
   default/wait-machine-pool-nodes: ["30m", "10s"]
+  csi-migration/wait-controlplane-upgrade: ["60m", "10s"]
+  csi-migration/wait-worker-nodes: ["60m", "10s"]
+  csi-migration/wait-control-plane: ["60m", "10s"]
```

Comment:
do we really need 60 minutes to wait for the cluster, worker nodes, and control planes? 20 minutes should be way enough no?

Reply:
also if we want to reuse the default/ times above we don't need to add a specific one for csi-migration

Reply:
Yes. Did not want to use the default ones. 

>do we really need 60 minutes to wait for the cluster, worker nodes, and control planes? 20 minutes should be way enough no?

I have seen that 30 mins was not enough for worker nodes to comes up especially during the upgrade due to rolling upgrade. 
I did couple of runs -- one time it took like 42 mins. So had set it to 60 mins just for csi-migration spec.

Reply:
All of the tests has 3 CP + 3 Worker node.

Reply:
ok makes sense